src/components/tabs/Tabs.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <div class="aui-tabs horizontal-tabs">
       <ul class="tabs-menu">
-        <li ref="item" class="menu-item" v-for="tab in tabs">
+        <li ref="item" class="menu-item" :style="styleWidth" v-for="(tab, key) in tabs" :key="key">
```

Comment:
What is the `:key="key"` for?

Reply:
I always use `v-bind:key` in `v-for` because I think this is the recommended usage. Check [here](https://vuejs.org/v2/guide/list.html#key)

>It is recommended to provide a key with v-for whenever possible, unless the iterated DOM content is simple, or you are intentionally relying on the default behavior for performance gains.

But if you want me to removed it I can do it  :+1:

Reply:
I forgot about this comment before merging. In fact, it is recommended, but if I understand Vue correctly, setting index value to the `key` parameter will have exactly no effect other than misguiding the warning. In other words, if you remove the tab from the middle, all the subsequent tabs (dom elements) will be patched as if there was no key provided. Anyway, it's merged already...

Reply:
Understood, thank you.